### ISSUE TITLE
Release notes?

### ISSUE THREAD

**Issue by hughperkins** [semver/semver#993]
I see there are multiple versions of semver, 1.0.0, 2.0.0, etc. I dont see any releases on this github repo, or release notes. Where can I find those? If they don't already exist, I feel it might be useful to create those? (and if they do, then maybe make them easier to find?)

**Comment by JohnTitor** [comment]
Yeah, AFAIK we don't have any changelog right now (and creating them for existing tags is somewhat hard).
See https://github.com/semver/semver/issues/976 for ambiguity after v2.0.0.

For better visibility, I've created GitHub Releases for existing tags: https://github.com/semver/semver/releases
Hope these help you check related commits!

**Comment by hughperkins** [comment]
Ok. I mean, I was hoping for more of a summary of what changed between 1.x.x and 2.0.0 for example, rather than simply a list of commits. I feel like it could be good for this repo to set a good example, considering this repo's topic is strongly related to releases :)

**Comment by hughperkins** [comment]
Note that if you copied the tags into branches, then it would be straight forward to use the github 'Compare' to compare the two versions.

**Comment by JohnTitor** [comment]
Note that the notes on GitHub Releases are auto-generated! I hope someone could write down some more things but I couldn't find any discussion or something else other than commits...

**Comment by jwdonahue** [comment]
@hughperkins I just posted some history notes on the 2.0.0 spec (https://github.com/semver/semver/issues/976#issuecomment-1867994508). WIP: going back further and into more detail.

**Comment by jwdonahue** [comment]
@hughperkins The full history, since mojombo handed everything off to haacked, can be found here: https://github.com/semver/semver/commits/master/semver.md

Changes tend to accumulated over here on semver/semver and then get dumped in batches into the official document(s) posted over on semver/semver.org.

**Comment by 66244484** [comment]
Good day requesting trassfroing app please with extra mapping guides in to dissection..
Thank you ..
Mo.b

**Comment by xxtrakyle** [comment]
Nothing to preview
